Units/parser-inko.r/let.d/input.inko

```diff
@@ -1,2 +1,3 @@
 let FOO = 10
 let BAR = 20
+let foo: Bla = 30
```

Comment:
Don't you want to tag `foo`?
I wonder why you don't have to update expected.tags.

Reply:
@masatake No, `foo` isn't supposed to be tagged. The bug was that in this case `Bla` _would_ be tagged, which is incorrect.

Reply:
Oh, I see.

Reply:
@masatake `foo` is a local variable, and they should not be tagged. They were not tagged either.

However, when defining a local variable with an explicit type (as is done in the above example), ctags would incorrectly generate a tag for `Bar`. The expected behaviour is no tags being generated for local variables ever.

Reply:
I see.